vsphere_plugin_common/__init__.py

```diff
@@ -707,10 +769,32 @@ def get_server_ip(self, vm, network_name):
                 )
                 return ip_address
 
-    def _wait_vm_running(self, task):
+    def _task_guest_state_is_running(self, task):
```

Comment:
This (& `_task_guest_has_networks`) are just masquerading as methods and don't use self. Could they be moved out of the class?